### PR TITLE
Navigation rework

### PR DESCRIPTION
**Pull request by bmallred**
We are doing multiple changes in one:
 - Make collapsable items in the navigation not redirect on click
 - Removing references to the `IntroHeader` component which provided the **Tour** and **Full Review** options
 - Adding a **Intro** and **Review** section to the main sections
 - Any **Intro** section which does not have clear text within the content guide we need to insert temporary placeholders for them
 - Adjust navigation on **prev** and **next** buttons per section view
 - Adjust validation logic to exclude the **intro** and **review** sections

Issues: #1525, #1526, #1527, #1528, #1529, #1530, #1531, #1532, #1533, #1534

### CODE REVIEW

**Comment by floresj** [comment]
Is this one good to review sir?

**Comment by bmallred** [comment]
It is!

File: src/components/Form/IntroHeader/IntroHeader.jsx [hunk 1]
```diff
@@ -3,6 +3,11 @@ import { i18n } from '../../../config'
 import Svg from '../Svg'
 
 export default class IntroHeader extends React.Component {
+  constructor (props) {
+    super(props)
+    console.warn('DEPRECATED: Remove IntroHeader once approved.')
```
Comment (floresj) [inline src/components/Form/IntroHeader/IntroHeader.jsx]: Deprecation notices === :heart:

**Comment by floresj** [comment]
# Update menu single instance
selection_304

**Comment by floresj** [comment]
#1525 Validated all sections have introduction and review sections. Introduction pages have placeholder content which will be updated later on.

**Pull request merged**